src/gui/mainwindow.cpp

```diff
@@ -14,6 +16,19 @@ static QString DefaultWindowTitle() noexcept
 	return "Neovim";
 }
 
+static void SetDefaultWindowSize(QWidget* widget)
+{
+	QPoint widget_position = widget->mapToGlobal(QPoint{ widget->width() / 2, 0 });
+	QScreen* screen = qApp->screenAt(widget_position);
+	QRect geometry{ screen->availableGeometry() };
```

Comment:
Needs a nullptr check, since `screenAt` can return null.

> Returns the screen at point, or nullptr if outside of any screen.

https://doc.qt.io/qt-6/qguiapplication.html#screenAt